R/lint.R

```diff
@@ -81,6 +81,13 @@ lint <- function(filename, linters = NULL, cache = FALSE, ..., parse_settings =
       else {
         expr_lints <- flatten_lints(linters[[linter]](expr)) # nolint
 
+        if (length(expr_lints)) {
+          expr_lints[] <- lapply(expr_lints, function(lint) {
```

Comment:
Is there any easy way to only do this for the linters that are named differently than "default"?

Reply:
Not that I'm aware of. However the linters list is auto-named so we're not breaking anything either.

Reply:
OK. I meant more as a way to avoid overwriting the whole list. Not crucial since the list is small.

Reply:
Could add a check on linter != expr_lints[[1L]]$linter. WDYT?

Reply:
ehhh let's leave it as is for now. That doesn't really buy us much.